uiServices/src/services.go

```diff
@@ -98,53 +132,88 @@ func validateJsonWithSchema(config []byte) bool {
 
 func getConfigs(rw http.ResponseWriter, req *http.Request){
 
-    configPathDir := req.Header.Get("configPathDir")
+    configPathDir := getConfigHeader(req)
     configName := chi.URLParam(req, "configName")
 
-    if !strings.HasSuffix(configPathDir, "/"){
-        configPathDir = configPathDir + "/"
-    }
-
-    if len(configPathDir) <= 1{
-        rw.WriteHeader(http.StatusBadRequest)
-        logrus.Error("No Header Path Found")
+    if !validateFileNameAndHeader(rw, req,configPathDir,configName){
         return
     }
 
-    file, err := os.Open(fmt.Sprintf("%s%s.xml", configPathDir, configName))
-    if err != nil {
-        logrus.Error("Configuration Name Not Found: "+configPathDir + configName)
-        rw.WriteHeader(http.StatusNotFound)
+        file, err := os.Open(fmt.Sprintf("%s%s.xml", configPathDir, configName))
+        if err != nil {
+            logrus.Error("Configuration Name Not Found: " + configPathDir + configName)
+            rw.WriteHeader(http.StatusNotFound)
+            return
+        }
+
+        defer file.Close()
+
+        var config perfTestUtils.Config
+
+        byteValue, err := ioutil.ReadAll(file)
+        if err != nil {
+            rw.WriteHeader(http.StatusInternalServerError)
+            logrus.Error("Cannot Read File")
+            return
+        }
+
+        err = xml.Unmarshal(byteValue, &config)
+        if err != nil {
+            rw.WriteHeader(http.StatusInternalServerError)
+            logrus.Error("Cannot Unmarshall")
+            return
+        }
+
+        configJson, err := json.MarshalIndent(config, "", "")
+        if err != nil {
+            rw.WriteHeader(http.StatusInternalServerError)
+            logrus.Error("Cannot Marshall")
+            return
+        }
+
+        rw.WriteHeader(http.StatusOK)
+        rw.Write(configJson)
+        logrus.Println(string(configJson))
+
+
+}
+
+func putConfigs(rw http.ResponseWriter, req *http.Request) {
```

Comment:
run `go fmt ./..`. at the automated-perf-test level

Reply:
Not getting any errors from go fmt

![image](https://user-images.githubusercontent.com/34065910/35681728-3830be28-0756-11e8-91d4-0f63f791678a.png)